### PR TITLE
Branch update ug

### LINKED ISSUE #262: UG: Command Parameters for gradedtest
image

Change `name` to graded test field

### LINKED ISSUE #265: UG: Save the victims of mass replace
image

### LINKED ISSUE #266: UG: Sidebar not blue :(
image

Fix style error

### LINKED ISSUE #267: UG: front 3 tables not same width
image

Fix this please future me, thanks!

### LINKED ISSUE #268: UG: markdown not converting to vertical bars
image

Fix this pls future me

### PR DESCRIPTION
**Pull request by raydenlim**
Fix table size
Fix div border color
Fix `&#124` not showing as `|` in markbind
Fix victim of mass replace 
image

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -226,16 +226,16 @@ The following describes what each tab displays on the **Current Panel**:
...
-    <md>
-    :warning: **Warning**:
-    
-    * This action is irreversible
-    </md> </div>
+<div class="alert alert-warning">
+<md>
+:warning: **Warning**:
+
+* This action is irreversible
```
Comment (Respirayson) [inline docs/UserGuide.md:233]: Good catch

File: docs/UserGuide.md [hunk 2]
```diff
@@ -226,16 +226,16 @@ The following describes what each tab displays on the **Current Panel**:
...
+  </md> </div>
 
 
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some 📖Example commands you can try:
+   Some Example commands you can try:
```
Comment (Respirayson) [inline docs/UserGuide.md]: actually should Example be capitalised?

**Review by Respirayson — APPROVED** [review]
LGTM, one small nit

File: docs/UserGuide.md (hunk 2 above)
Comment (raydenlim) [reply docs/UserGuide.md]: good catch, will update this

**Pull request merged**